openam-authentication/openam-auth-ldap/src/main/resources/amAuthLDAP.properties

```diff
@@ -26,6 +26,7 @@
 #
 # Portions Copyrighted 2011-2016 ForgeRock AS.
 # Portions Copyrighted 2012 Open Source Solution Technology Corporation
+# Portions Copyrighted 2019 Open Source Solution Technology Corporation
```

Comment:
`2019` -> `2012-2019`. 
And please delete previous line.

Reply:
IC.